iOS/G3MiOSSDK/Commons/Basic/ElevationDataProvider.hpp

```diff
@@ -68,7 +68,13 @@ class ElevationDataProvider {
                                                const Vector2I& extent,
                                                IElevationDataListener* listener,
                                                bool autodeleteListener) = 0;
-
+    
+  virtual const long long requestElevationData(const Sector& sector,
+                                               int level, int row, int column,
+                                                 const Vector2I& extent,
+                                                 IElevationDataListener* listener,
+                                                 bool autodeleteListener) = 0;
+    
   virtual void cancelRequest(const long long requestId) = 0;
```

Comment:
I'm wondering if the right thing to do is to define just one requestElevationData() that receives by parameter the involved Tile.